lib/jira.py

```diff
@@ -22,11 +22,38 @@
 """
 
 import argparse
+import base64
 import json
 import os
+import sys
+import subprocess
 import requests
 
 
+def read_json(file):
+    if os.path.getsize(file) > 0:
+        with open(file, 'r') as f:
+            data = json.load(f)
+        return data
+
+
+def get_release_namespace(data_release):
+    return data_release['metadata']['namespace']
+
+
+def get_namespace_from_release(release):
+
+    data_release = read_json(release)
+
+    if not data_release:
+        log(f"Empty release file {release}")
+        exit(0)
+
+    ns = get_release_namespace(data_release)
+    log(f"Namespace: {ns}")
+    return ns
```

Comment:
If the parameter for this method is a file name, the parameter name should reflect that.

```suggestion
def get_namespace_from_release(release_file_name):

    data_release = read_json(release_file_name)

    if not data_release:
        log(f"Empty release file {release_file_name}")
        exit(0)

    ns = get_release_namespace(data_release)
    log(f"Namespace extracted from file {release_file_name}: {ns}")
    return ns
```

Reply:
fixed